CONTRIBUTING.md

```diff
@@ -1,24 +1,62 @@
 # How to contribute to this project ?
 
-1. First you must choose an issue on [the project github](https://github.com/mdl29/scratchy). If you want to do something but no issue is created yet you must create one (try to tag it properly).
-2. Fork the project.
-3. Create a new branch whith the following name format on your fork : `resolves #ISSUEID Component: title for features/fix`.
+## **Choose a issue :**
 
-> Example : if you want to solve this issue (image below) you should have named your branch `resolvec #138 tool : kickstarter for yannis`.
-> Component are : backend / frontend / tools / doc
-> ![example issue](./doc/example_issue.png)
+You must choose an issue on [the project github](https://github.com/mdl29/scratchy). If you want to do something but no issue is created yet you must create one (try to tag it properly).
```

Comment:
Maybe add link to the issue page  ? simply add a `/issues` add the end of the link